README.md

```diff
@@ -364,3 +364,30 @@ Escapes a string's HTML. Specifically, it makes these replacements:
 - " (double quote) is converted to &quot;
 - & is converted to &amp;
 
+### Macros
+Macros are a simple way to re-use template bits. Think of them as functions that you can call
+and that returns some text.
+
+Macros are defined the following way:
+
+```jinja2
+{% macro hello_world(greeting, language) %}
```

Comment:
This is a bit of a weird example since it won't emit anything if language is not equal to "en". Maybe you can add some "else if"s or maybe just an "else"?